assets/src/scss/layout/_navbar.scss

```diff
@@ -214,60 +215,43 @@ $navbar-default-height: 60px;
   }
 }
 
-:root {
-  --nav-link--color: $white;
-  --nav-link--opacity: 1;
-  --nav-link--padding: #{$sp-2} 0;
-  --nav-link-active--color: var(--nav-link--color, #{$white});
-  --nav-link-active--opacity: var(--nav-link--opacity, 1);
-  --nav-link-active--text-decoration-color: #{$gp-green};
-  --nav-link-active--text-decoration-line: underline;
-  --nav-link-active--text-decoration-style: solid;
-  --nav-link-active--text-decoration-thickness: 3px;
-  --nav-link-active--text-underline-offset: auto;
-  --nav-link-active--text-underline-position: under;
-  --nav-link--hover--color: $white;
-  --nav-link--hover--opacity: 1;
-  --nav-link--hover--text-decoration-color: #{$gp-green};
-  --nav-link--hover--text-decoration-line: underline;
-  --nav-link--hover--text-decoration-style: solid;
-  --nav-link--hover--text-decoration-thickness: 3px;
-  --nav-link--hover--text-underline-offset: auto;
-  --nav-link--hover--text-underline-position: under;
-}
-
 a.nav-link {
-  color: var(--nav-link--color);
+  --nav-link-- {
+    color: $white;
+
+    &:hover {
+      color: $white;
+      text-decoration-color: $gp-green;
+    }
+  }
+
+  opacity: 1;
   display: inline-block;
-  opacity: var(--nav-link--opacity);
-  padding: var(--nav-link--padding);
+  padding: $sp-2 0;
   position: relative;
   width: 100%;
 
   @include large-and-up {
-    --nav-link--padding: 0;
-    --nav-link-active--text-decoration-thickness: 4px;
```

Comment:
This one moved [here](https://github.com/greenpeace/planet4-master-theme/pull/1627/files#diff-91009267fa7864d1b148d657a615793b252f3e148c4001b639eab5a4c51324e4R281).